framework/py/flwr/serverapp/strategy/strategy.py

```diff
@@ -271,5 +272,9 @@ def start(
         log(INFO, "")
         log(INFO, "Strategy execution finished in %.2fs", time.time() - t_start)
         log(INFO, "")
+        log(INFO, "Final results:")
+        for line in io.StringIO(str(result)):
+            log(INFO, "\t%s", line.strip("\n"))
```

Comment:
Just a quick question, not a suggestion.
What's the difference between `split` and `StringIO`? Is the `StringIO` more efficient or memory-friendly?

```suggestion
        for line in str(result).split("\n"):
            log(INFO, "\t%s", line)
```

Reply:
should be more memory efficient if `results` is super large. We might not hit that point unless there are thousands of rounds with many metrics computed. Happy to replace with `.split`. I mostly borrowed this logic from https://github.com/adap/flower/blob/19cbbb7872137b3627f1210c408b2bf38ca8b4d7/framework/py/flwr/server/server.py#L499